requirements/documentation.txt

```diff
@@ -0,0 +1,8 @@
+# Documentation (for devs)
+# -----------------------
+
```

Comment:
@jmkerloch you need to update requirements to use `furo` instead of `sphinx-book-theme`